content/applications/sales/rental.rst

```diff
@@ -60,44 +131,146 @@ Odoo always uses two rules to compute the price of a product when you create a r
-customers before they pick up products to make sure your products are returned on time and in their
-original condition. To do so, go to :menuselection:`Rental --> Configuration --> Settings`, activate
-*Digital Documents*, and *Save*.
+Upon confirming a rental order, the ability to request the customer sign a rental agreement,
+outlining the arrangement between the company and customer, *before* they pick up the rental
+product(s) appears via a :guilabel:`Sign Documents` button.
 
-.. image:: rental/digital-documents-settings.png
-   :align: center
-   :alt: Digital Documents settings in Odoo Rental
+Such documents can ensure everything is returned on-time and in their original condition, becomes
+available.
+
+.. important::
+   The :guilabel:`Sign Documents` button/option **only** appears if the :guilabel:`Digital
+   Documents` feature has been activated in the *Rental* application settings. To do so, navigate to
+   :menuselection:`Rental app --> Configuration --> Settings`, activate :guilabel:`Digital
+   Documents`, and click :guilabel:`Save`.
 
 .. note::
-   This feature requires the :doc:`Sign <../finance/sign>` app. If necessary, Odoo installs it after
-   activating *Digital Documents*.
+   This feature also requires the :doc:`Sign <../finance/sign>` app. If necessary, Odoo installs it
+   after activating the :guilabel:`Digital Documents` setting.
+
+To request a customer signature of this nature, select a confirmed rental order, click the
+:guilabel:`Sign Documents` button to reveal a :guilabel:`Sign Documents` pop-up window.
 
-Once the app settings are saved, you have the option to change the default *Rental Agreement* from
-the dropdown menu. You can pick any document already uploaded to the *Sign* app, or upload a new one
-to the *Sign* app by clicking on *Upload Template*.
+.. image:: rental/sign-documents-popup.png
+   :align: center
+   :alt: The Sign Documents pop-up window that appears in the Odoo Rental application.
+
+From here, select the desired document from the :guilabel:`Document Template` field. Then, click
+:guilabel:`Sign Document`. Doing so reveals a :guilabel:`New signature Request` pop-up window.
+
+.. image:: rental/new-signature-request-form.png
+   :align: center
+   :alt: The New Signature Request pop-up window that appears in the Odoo Rental application.
 
-To request a customer signature, select a confirmed rental order, click on *Sign Documents*, choose
-the document template and click on *Sign Documents* again. On the next window, select your customer
-and click on *Sign Now* to start the signing process with your customer. Once the document is
-completed, click on *Validate & Send Completed Document*.
+Upon confirming the information in the :guilabel:`New Signature Request` pop-up form, click
+:guilabel:`Sign Now` to initiate the signing process.
+
+A separate page is then revealed, showcasing the document to be signed. Odoo guides the customer
```

Comment:
It's my understanding that it can send it to the customer to sign -- and it would accessible to them via the Customer Portal. I'll add a mention of this to make things a bit more clear.